### PR TITLE
Start to stub in categorize section

### PR DESCRIPTION
**Pull request by cgreene**
Includes chunks that someone can bite off if they want to focus on a specific area for writing.

### CODE REVIEW

**Comment by cgreene** [comment]
@agitter : totally agree that italics for prompts and suggestions would be most helpful. I'll convert to that (and fix a couple of the markdown issues that CC noted). Thanks!

**Comment by agitter** [comment]
I added a placeholder for background information on deep learning, which came up in #116.

**Comment by cgreene** [comment]
@agitter : ok - tried to address. Some of the codeclimate issues will take care of themselves as we write and clarify our thoughts (e.g. header length/punctuation). Basically these are all stubs to be replaced. Thanks!

File: sections/02_intro.md [hunk 1]
```diff
@@ -1,3 +1,25 @@
...
+care systems are extensively tracked. These sources of information are combined
+to help to guide our health care and maintenance. (CATEGORIZE) We'd be able to
+compare our state and trajectory to (anonymized) others, and identify means to
+improve our health. (TREAT) These means might contain drug combinations selected
+based on personalized predictions.
+
+### If this happens, is deep learning required for any of it? Are we any closer
+### because of the advent of deep learning?
+
+* "Categorize" and "treat" sound a bit like PMI goals. Another way to think
```
Comment (gwaygenomics) [inline sections/02_intro.md]: I don't think you meant to bullet this?

Comment (gwaygenomics) [inline sections/02_intro.md]: also, can you expand the acronym for PMI the first time you reference it?

File: sections/02_intro.md [hunk 2]
```diff
@@ -1,3 +1,25 @@
 ## Introduction
 
-Intro goes here.
+### Potential writing prompt
+
+One potential future that we could imagine is a world in which data, once
```
Comment (gwaygenomics) [inline sections/02_intro.md]: I definitely agree with the three subcategories and the content of this stub. 

But, for consistency, can you mention them either before or after the period? Right now I am not sure which sentences `(CATEGORIZE)` or `(TREAT)` are referencing

**Comment by agitter** [comment]
My comments have been addressed

File: sections/02_intro.md (hunk 2 above)
Comment (cgreene) [inline sections/02_intro.md]: @gwaygenomics : removed them for now. We'll see how the sections come along to decide what to talk about specifically.

File: sections/02_intro.md (hunk 1 above)
Comment (cgreene) [inline sections/02_intro.md]: Bullets here (and elsewhere) seem to be induced by atom's convenient "reflow selection" functionality. I tracked it down here and in the other part, and expanded PMI. Thanks!

**Comment by cgreene** [comment]
@gwaygenomics : let me know if I've addressed all of your comments! thanks!

**Pull request merged**